engine/api/src/main/java/pl/touk/nussknacker/engine/api/editor/DefaultEditorMode.java

```diff
@@ -0,0 +1,5 @@
+package pl.touk.nussknacker.engine.api.editor;
+
+public enum DefaultEditorMode {
```

Comment:
I think that it should be named `DualEditorMode` which is used to determine `defaultMode` for `DualEditor`. WDYT?

Reply:
Done